<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.springframework.test.web.servlet.samples.standalone.ExceptionHandlerTests$MvcTests" tests="3" skipped="0" failures="0" errors="0" timestamp="2021-06-20T08:12:31" hostname="gankangdembp" time="0.24">
  <properties/>
  <testcase name="localExceptionHandlerMethod()" classname="org.springframework.test.web.servlet.samples.standalone.ExceptionHandlerTests$MvcTests" time="0.072"/>
  <testcase name="globalExceptionHandlerMethodUsingClassArgument()" classname="org.springframework.test.web.servlet.samples.standalone.ExceptionHandlerTests$MvcTests" time="0.095"/>
  <testcase name="globalExceptionHandlerMethod()" classname="org.springframework.test.web.servlet.samples.standalone.ExceptionHandlerTests$MvcTests" time="0.073"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
